apps/backend/src/web/middlewares/tokenless-strategies/github-actions.ts

```diff
@@ -51,6 +51,7 @@ const strategy = {
       .withGraphJoined("[activeInstallation, projects]")
       .where("githubAccount.login", authData.owner)
       .findOne("github_repositories.name", authData.repository)
+      .orderBy("github_repositories.githubId", "desc")
```

Comment:
Not sure it works if I rename an old repository.

Reply:
You can't rename an old repo. It has been deleted on GitHub